packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -0,0 +1,82 @@
+import { forwardRef, HTMLAttributes, MouseEvent } from "react";
+import { clsx } from "clsx";
+import { makePrefixer, useId } from "@salt-ds/core";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listItemNextCss from "./ListItemNext.css";
+import { useListItems } from "./ListNextContext";
+
+const withBaseName = makePrefixer("saltListItemNext");
+
+export interface ListItemNextProps extends HTMLAttributes<HTMLLIElement> {
+  label?: string;
+  /**
+   * If true, the particular list item in list will be disabled.
+   */
+  disabled?: boolean;
+  selected?: boolean;
+  id?: string;
+  value: string;
+}
+
+export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
+  function ListItemNext(
+    {
+      children,
+      className,
+      disabled: disabledProp,
+      selected: selectedProp,
+      label,
+      id: idProp,
+      value,
+      onClick,
+      ...props
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-item-next",
+      css: listItemNextCss,
+      window: targetWindow,
+    });
+
+    const listContext = useListItems();
+    const id = useId(idProp) || value;
+    const itemId = `list-${listContext?.id || "listNext"}--list-item--${id}`;
```

Comment:
think we could have something other than "listNext" here (maybe another default id? the one above?) because otherwise when it changes to list from list next, it would be `list-list--...`

Reply:
I agree. I also believe we don't need to add any other words if there is already an id provided. So something like `${listContext?.id || "listNext"}-${id}` makes more sense to me